doc/replacements.rst

```diff
@@ -0,0 +1,6 @@
+.. backslash followed by zero-width space; to help Windows paths wrap better
+.. |backslash| unicode:: 0x5C U+200B
```

Comment:
BTW, I'm not sure this is entirely necessary... it does cause wrapping to happen *after* the `\` rather than before, but I believe I added this originally because `parsed-literal` doesn't wrap by default, but the fix for that is CSS, not zero-width spaces. (OTOH, this with `:trim:` allows a path to span multiple lines in the source, which is probably a nice benefit. I suppose for that reason I'd be inclined to keep it, though the zwsp could be dropped if we think wrapping before the `\` is better.)